src/HomePage.js

```diff
@@ -1,12 +1,56 @@
 import React from 'react';
+import { Card } from 'react-bootstrap';
+import { Accordion } from 'react-bootstrap';
+import { Navbar } from 'react-bootstrap';
+import { Nav } from 'react-bootstrap';
+import { Container } from 'react-bootstrap';
```

Comment:
This can be combined into `import { Card, Accordion, Navbar, Nav, Container} from 'react-bootstrap';`